doc/user/command-line.xml

```diff
@@ -350,7 +363,7 @@ foo.in
 
       <para>
 
-      But if the <varname>$NUM_CPU</varname>
+      But if the <envar>NUM_CPU</envar>
```

Comment:
does <envvar> prefix it with $?

Reply:
I rather think the `$` reference in 345 should be removed.  Generally speaking, the names of variables are FOO, not $FOO.  It makes some sense to include the `$` if it's shown in a context where you're expecting subst activity but in this context it's actually talking about an external (os.environ) value, not even a construction var, so that's not the case.  And yes, this is pointless nitpickery on my part...